src/servers/src/http/prometheus.rs

```diff
@@ -690,6 +690,41 @@ pub async fn label_values_query(
         };
         table_names.sort_unstable();
         return PrometheusJsonResponse::success(PrometheusResponse::LabelValues(table_names));
+    } else if label_name == FIELD_NAME_LABEL {
+        let mut field_columns = HashSet::new();
+        let mut matches = params.matches.0;
+        // query all tables if no matcher is provided
+        if matches.is_empty() {
+            matches = match handler
+                .catalog_manager()
+                .table_names(&catalog, &schema)
```

Comment:
We have `tables` API for fetching all tables info : https://h6tbhdl56ey8.cn-hangzhou.alicloud.cloud.greptime.cn/playground/